### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jjasghar**
More pods and services have been added since this doc has been written.

### CODE REVIEW

**Comment by linsun** [comment]
I think some of these are optional components that exist in demo by default but not in the helm install by default.  it would be good to put some wording around it to avoid confusion.

**Comment by jjasghar** [comment]
I signed it!

**Comment by jjasghar** [comment]
@linsun absolutely. But if you take a look https://github.com/istio/istio.io/pull/2750/files#diff-53d1c76fbb0d32483d8d7659df27fe7aL60 it assumes you are running the demo scripts and all not "just istio." 

I was doing the commands verbatim, and there were a difference in what I saw and what the doc said.

**Comment by jjasghar** [comment]
Should we put two verification sections? That might get noisy? I dunno.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 33bbc6ced9770c34efa55d51fcbfc4b016c78f74

https://app.netlify.com/sites/preliminary-istio/deploys/5bbd0833c96592718f18bebc

**Comment by jjasghar** [comment]
Added some more verbosity about the difference between the `demo.yaml` and the `helm` installation.

**Comment by jjasghar** [comment]
Bump @linsun :metal:

**Comment by rcaballeromx** [comment]
@jjasghar Big picture question: Why are we including the commands' outputs? Is there anything in particular the reader needs to verify on them? If that is not the case, just get rid of the outputs and make the document a lot easier to maintain.

**Comment by jjasghar** [comment]
@rcaballeromx i believe i got the changes done how you'd like em.

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 1]
```diff
@@ -118,17 +106,6 @@ Follow our instructions on how to
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start/index.md]: Tricky, I know, but if you remove the output you need to change the text in the step too.
1.  Ensure the corresponding Kubernetes pods are deployed and the status of the following containers
     is shown as `Running`: `istio-pilot-*`, `istio-ingressgateway-*`,
     `istio-egressgateway-*`, `istio-policy-*`, `istio-telemetry-*`,
     `istio-citadel-*`, `prometheus-*`, `istio-galley-*`, and, optionally,
     `istio-sidecar-injector-*`.

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 2]
```diff
@@ -90,18 +90,6 @@ Follow our instructions on how to
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start/index.md]: Similarly here, you have to update the text in the step if you remove the output:
 1.  To ensure the following Kubernetes services are deployed: `istio-pilot`,
     `istio-ingressgateway`, `istio-policy`, `istio-telemetry`, `prometheus`,
     `istio-galley`, and, optionally, `istio-sidecar-injector`, verify they all have an appropriate `CLUSTER-IP`:

Comment (jjasghar) [reply content/docs/setup/kubernetes/quick-start/index.md]: Fixed :metal:

File: content/docs/setup/kubernetes/quick-start/index.md (hunk 1 above)
Comment (jjasghar) [reply content/docs/setup/kubernetes/quick-start/index.md]: Fixed :metal:

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 3]
```diff
@@ -98,11 +100,11 @@ Follow our instructions on how to
...
 
-1.  Ensure the corresponding Kubernetes pods are deployed and all containers
-    are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
-    `istio-egressgateway-*`, `istio-policy-*`, `istio-telemetry-*`,
-    `istio-citadel-*`, `prometheus-*`, `istio-galley-*`, and, optionally,
-    `istio-sidecar-injector-*`.
+1.  Ensure the corresponding Kubernetes pods are deployed and all containers: `istio-citadel-*`,
+    `istio-engressgateway-*`, `istio-galley-*`, `istio-ingress-*`, `istio-ingressgateway-*`,
+    `istio-pilot-*`, `ingress-policy-*`, `ingress-statsd-prom-bridge-*`, `ingress-telemetry-*`,
+    `prometheus-*`, and optionally, `istio-sidecar-injector-*`:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start/index.md]: You are missing a verb:
and, optionally, `istio-sidecar-injector-*` have

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 4]
```diff
@@ -110,25 +100,13 @@ Follow our instructions on how to
...
 
-1.  Ensure the corresponding Kubernetes pods are deployed and all containers
-    are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
-    `istio-egressgateway-*`, `istio-policy-*`, `istio-telemetry-*`,
-    `istio-citadel-*`, `prometheus-*`, `istio-galley-*`, and, optionally,
-    `istio-sidecar-injector-*`.
+1.  Ensure the corresponding Kubernetes pods are deployed and all containers: `istio-citadel-*`,
+    `istio-engressgateway-*`, `istio-galley-*`, `istio-ingress-*`, `istio-ingressgateway-*`,
+    `istio-pilot-*`, `istio-policy-*`, `istio-statsd-prom-bridge-*`, `istio-telemetry-*`,
+    `prometheus-*`, and optionally have `istio-sidecar-injector-*`:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start/index.md]: Grammar indeed. Pay close attention to the commas and the order of the words, your fix is not correct. The correct text in full: 
1. Ensure the corresponding Kubernetes pods are deployed and all containers, `istio-citadel-*`,
    `istio-engressgateway-*`, `istio-galley-*`, `istio-ingress-*`, `istio-ingressgateway-*`,
    `istio-pilot-*`, `istio-policy-*`, `istio-statsd-prom-bridge-*`, `istio-telemetry-*`,
    `prometheus-*`, and, optionally, `istio-sidecar-injector-*`, have a `STATUS`of `Running`:

**Pull request merged**